cdcsdk-testing/src/test/java/com/yugabyte/cdcsdk/testing/PostgresSinkConsumerIT.java

```diff
@@ -136,7 +137,12 @@ public void verifyRecordsInKafka() throws Exception {
     public void verifyRecordsInPostgresFromKafka() throws Exception {
         // Adding Thread.sleep() here because apparently Awaitility didn't seem to work as expected.
         // TODO Vaibhav: Replace the Thread.sleep() function with Awaitility
-        Thread.sleep(10000);
+        // Thread.sleep(10000);
```

Comment:
Can you remove the comment before committing?

Reply:
Yes. I'll remove them